internal/iam/command.go

```diff
@@ -39,3 +42,49 @@ func New(cfg *config.Config, prerunner pcmd.PreRunner) *cobra.Command {
 
 	return cmd
 }
+
+func AddResourceOwnerFlag(cmd *cobra.Command, cliCommand *pcmd.AuthenticatedCLICommand) {
```

Comment:
Can be private since it's only used in this package.
```suggestion
func addResourceOwnerFlag(cmd *cobra.Command, cliCommand *pcmd.AuthenticatedCLICommand) {
```